src/tar_check.rs

```diff
@@ -112,16 +112,24 @@ fn tar_check_archive<R: Read>(mut archive: Archive<R>, path_str: &str) {
 			eprintln!("Package {} has no SUID files.", path_str);
 		}
 		eprint!(
-			"[E]=list executable files, [L]=list all files, \
-			 [T]=run shell to inspect, "
+			"{}{}, ",
+			"[E]".bold().green(),
+			"=list executable files".green()
 		);
+		eprint!("{}{}, ", "[L]".bold().green(), "=list all files".green());
+		eprint!(
+			"{}{}, ",
+			"[T]".bold().cyan(),
+			"=run shell to inspect".cyan()
+		);
+
 		if has_install {
-			eprint!("{}", "[I]=show install file, ".bright_red().bold());
+			eprint!("{}, ", "[I]=show install file".bright_red().bold());
```

Comment:
Hey, here I'm somewhat unsure. The install file itself could be dangerous, but showing the file is not a dangerous operation.

Should we change it to something like `[I]=show [red]install file[/red]` ? Or something else?

Alternatively, make all actions except installation to be slightly green or even neutral? As far as my preferences go, I think I'd consider making everything neutral except `install file` text and `ok, proceed` -- because those are the only things or operations that "could harm"

Reply:
I'll give that a try.

Reply:
Is this what you had in mind?

<img width="685" alt="image" src="https://user-images.githubusercontent.com/137616/120883946-edf0d500-c5e8-11eb-81e7-2cc6e5cf8a05.png">